spec/fabricators/user_fabricator.rb

```diff
@@ -103,9 +107,8 @@
 end
 
 Fabricator(:trust_level_0, from: :user) { trust_level TrustLevel[0] }
-
 Fabricator(:trust_level_1, from: :user) { trust_level TrustLevel[1] }
-
+Fabricator(:trust_level_2, from: :user) { trust_level TrustLevel[2] }
```

Comment:
Not sure why this guy didn't get his own fabricator. Is TL2 the least significant trust level? 😛

Reply:
Always forgotten and pushed aside by his more (or less) trustworthy brothers, sad